### DISCUSSION TITLE
Please help me understand correct setting for good photography outputs, no other styles.

### DISCUSSION THREAD

**Discussion by apaicu123** [TheLastBen/fast-stable-diffusion#534] [Q&A]
Hi, Ben and awesome comunity, 

I love your work, I have been spending the last month training several models and trying to figure it out. 

I am trying to generate fashion photography using photos of one person. I am not interested in drawings or illustrations. Just good quality clothing options for the same person.

I have around 100 instance pics, some good quality, some average scraped from social media.

Please help me understand the correct setting for good photography outputs that reflect this person's body and face, with various clothing options.

Thank you in advance, cheers

**Comment by TheLastBen** [comment]
Hi,
If you want to try different -custom- clothing objects on a single human subject, you simply create 20 instance images for the person, and 5 or 10 instance images for each clothing object, and name the instance images something like this : 

Human subject images : hmmnnsbjt (1).jpg, hmmnnsbjt (2).jpg ...etc

Clothing objects : 
- cltthhhbj suit (1).jpg, cltthhhbj suit (2).jpg ...etc
- cddcltthhhbbj pants (1) cddcltthhhbbj  pants (2).... etc

put all the files in one folder and upload the as the instance images. for a total of 25 images, set the steps to 5000 and the text encoder to 40%

**Reply by apaicu123** [reply]
Hi, wow, thaks for replying so quickly, 

So let me make sure that I understand. 

I will start from zero, training a new model. I've got my pictures of the person, say 10 closeup face, 10 medium shot, 10 full body. 

_If you want to try different -custom- clothing objects on a single human subject, you simply create 20 instance images for the person, and 5 or 10 instance images for each clothing object,_

You mean I use my own 20 instance images or I generate them in Dreambooth?
Same question, I generate 10 dresses that I like, on any human subject?

Also, to be sure, you mean train from scratch or retrain the model on this new instance images?

**Reply by TheLastBen** [reply]
> I will start from zero, training a new model. I've got my pictures of the person, say 10 closeup face, 10 medium shot, 10 full body.

Correct

> You mean I use my own 20 instance images or I generate them in Dreambooth?

the instance images a real pictures of yourself and the real clothing object, not generated

> Also, to be sure, you mean train from scratch or retrain the model on this new instance images?

Yes you start from scratch, though you can train on subject at a time, but it is preferable to train all at the same time

**Reply by apaicu123** [reply]
OOk, now i get it. I did not express my need clearly. 
We do not have the real clothing, we need generated options. So there is no custom clothing, we just need options on a single human model. Same person, various poses, just photography outputs.

**Reply by TheLastBen** [reply]
that would be an easier task, just train on the human subject, then in the prompt, you can put whatever clothing you want on it.
